scripts-available/CDB_RegenerateTable.sql

```diff
@@ -0,0 +1,82 @@
+--
+-- Given a table, returns a series of queries that can be used to recreate it
+-- It does not include data
+--
+CREATE OR REPLACE FUNCTION @extschema@.__CDB_RegenerateTable_Get_Commands(tableoid OID)
+RETURNS text[]
+AS $$
+    import subprocess
+    import re
+
+    query = "SELECT current_database()::text as dname"
+    rv = plpy.execute(query, 1)
+    database_name_string = str(rv[0]['dname'])
+
+    query = """SELECT concat(quote_ident(nspname), '.', quote_ident(relname)) as quoted_name
```

Comment:
why using a query rather than relying on fomat %I and casting the tableoid to regclass? oh, uhm, I guess that would depend on the search_path to include or not the schema... and now that I think of it, using format from plpython means running a query anyway... oh, forget it

Reply:
Yeah, I wasn't sure how to do this in a more proper way but I do expect one to exist :D I just knew that doing `quote_ident(schema) + '.' + quote_ident(tablename)` should always work, independently of search_path or special characters in either of the names.